vagrant-tools/run-tests.sh

```diff
@@ -0,0 +1,99 @@
+. ~/.bash_profile
+cd /vagrant/borg/borg
+. ../borg-env/bin/activate
+if which pyenv > /dev/null; then
+  # for testing, use the earliest point releases of the supported python versions:
+  pyenv global 3.4.0 3.5.0
+fi
+# otherwise: just use the system python
+
+testingPartitionSize=128M
```

Comment:
I've switched it back to 16M with a custom value of 128M for NTFS because it still runs out at 32M.